Messaging.md

```diff
@@ -103,19 +137,19 @@ The following DLC-specific types are used throughout the specification. All type
 
 This type contains information about a contract's outcomes, their corresponding payouts, and the oracles to be used.
 
-#### Version 0 `contract_info`
+#### `single_contract_info`
 
-1. type: 55342 (`contract_info_v0`)
+1. type: 1
```

Comment:
Mmmm I feel `extends` is not really proper because there is no super type that is extended, rather there are multiple versions of the same field. I went with `implements` as it feels a bit more correct but let me know if you have a better idea.

Reply:
good by me